### PR TITLE
fix(protocol-designer): fix navbar z-index issue

### PR DESCRIPTION
**Pull request by koji**
# Overview
fix navbar z-index issue

Screenshot 2024-11-20 at 12 17 56 PM
Screenshot 2024-11-20 at 12 17 07 PM

Screenshot 2024-11-20 at 3 39 15 PM

close RQA-3641

### CODE REVIEW

**Comment by jerader** [comment]
one more thing is a bit buggy, i can click on the continue button in the toolbox

Screenshot 2024-11-20 at 12 31 27

**Comment by koji** [comment]
> one more thing is a bit buggy, i can click on the continue button in the toolbox
> 
> Screenshot 2024-11-20 at 12 31 27

@jerader 

this comes from Toolbox's `zIndex={3}`. 
Do you remember why you added zIndex or took from Slideout component?

File: components/src/organisms/Toolbox/index.tsx [hunk 1]
```diff
@@ -140,7 +140,6 @@ export function Toolbox(props: ToolboxProps): JSX.Element {
           <Box
             padding={SPACING.spacing16}
             boxShadow={isScrolledToBottom ? 'none' : '0px -4px 12px #0000001a'}
-            zIndex={3}
```
Comment (koji) [inline components/src/organisms/Toolbox/index.tsx:143]: Since the ToolBox has z-index: 10 set, I think it should be fine to remove it.

**Comment by jerader** [comment]
> > one more thing is a bit buggy, i can click on the continue button in the toolbox
> > Screenshot 2024-11-20 at 12 31 27
> 
> @jerader
> 
> this comes from Toolbox's `zIndex={3}`. Do you remember why you added zIndex or took from Slideout component?

@koji no idea, haha. if everything is fine with these changes, then i feel comfortable
 making them!

**Review by jerader — APPROVED** [review]
lgtm, thanks for fixing this

**Pull request merged**